src/workloads/execution/TimeSeriesSort.yml

```diff
@@ -0,0 +1,512 @@
+      OperationCommand:
+        createIndexes: *coll
+        indexes:
+        - key: {m: 1, t: 1}
+          name: MetaTime
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Operations:
+    - OperationMetricsName: DropMetaTimeIndex
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: *coll
+        index: MetaTime
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+  - *Nop
+  - Repeat: 1
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 103680
+    BatchSize: *batchSize
+    Document:
+      t: {^IncDate: {start: "2022-01-01", step: 400}}
+      m: {^RandomInt: {min: 0, max: 1000}}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 5000
+    BatchSize: *batchSize
+    Document:
+      t: {^IncDate: {start: "2022-01-01", step: 400}}
+      m: 1001
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10
+    BatchSize: *batchSize
+    Document:
+      t: {^IncDate: {start: "2024-01-01", step: 400}}
+      m: 1002
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: 1
+    Database: *db
+    Collection: *coll
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 503680
+    BatchSize: *batchSize
+    Document:
+      t: {^IncDate: {start: "2023-01-01", step: 400}}
+      m: {^RandomInt: {min: 0, max: 1000}}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: Queries
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - Repeat: &numQueryRuns 50
+    Database: *db
+    Operations:
+    - OperationMetricsName: SortQueryBlockingSort
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: *coll
+        pipeline: [{$sort: {t: 1}}]
+        cursor: {batchSize: *batchSize}
+  - Repeat: *numQueryRuns
+    Database: *db
+    Operations:
+    - OperationMetricsName: SortQueryBoundedSorter
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: system.buckets.Collection0
+        pipeline: [{$sort: {"control.min.t": 1}},
```

Comment:
Currently this $sort will do a blocking sort, because the query planner doesn't realize that _id contains control.min.time.  So a more realistic test would be to remove the $sort stage and add `hint: {$natural: 1}` to the command.  Then it will scan the collection in {_id: 1} order, unpack, bounded-sort.